rosbag2_storage/include/rosbag2_storage/metadata_io.hpp

```diff
@@ -12,25 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef ROSBAG2_STORAGE__STORAGE_INTERFACES__BASE_INFO_INTERFACE_HPP_
-#define ROSBAG2_STORAGE__STORAGE_INTERFACES__BASE_INFO_INTERFACE_HPP_
+#ifndef ROSBAG2_STORAGE__METADATA_IO_HPP_
```

Comment:
Why this change? What's the problem with `base_info_interface`? Remember that these interfaces were designed to be composable for many specific interfaces.

Reply:
See other comment referring to this change.